src/nnet3bin/nnet3-copy.cc

```diff
@@ -61,9 +65,43 @@ int main(int argc, char *argv[]) {
     
     Nnet nnet;
     ReadKaldiObject(raw_nnet_rxfilename, &nnet);
-    
+     
     if (learning_rate >= 0)
       SetLearningRate(learning_rate, &nnet);
+    if (!add_output_nodes.empty()) {
+      std::vector<string> output_nodes;
+      SplitStringToVector(add_output_nodes, ",", true, &output_nodes);
+      int32 num_new_outputs = output_nodes.size();
+      for (int32 ind = 0; ind < num_new_outputs; ind++) {
+        std::vector<string> add_output_name;
+        SplitStringToVector(output_nodes[ind], "/", true, &add_output_name);
+        KALDI_ASSERT(add_output_name.size() == 2);
+        //nnet.AddNodeName(add_output_name[0], add_output_name[1]);
+      }
+    }
+
+    if (!rename_node_names.empty()) {
+      std::vector<string> orig_names, 
+        new_names;
+      //GetRenameNodeNames(rename_node_names, &orig_names, new_names);
```

Comment:
I think making this a function (appropriately documented!) in nnet-utils.h called
 void RenameNodes(std::string &rename_node_names,  Nnet *nnet);
would make this much nicer.